src/tests/class_tests/openms/data/MRMDecoyGenerator_output.TraML

```diff
@@ -465,12 +455,51 @@
       <cvParam cvRef="MS" accession="MS:1001226" name="product ion intensity" value="55.7"/>
       <cvParam cvRef="MS" accession="MS:1002008" name="decoy SRM transition"/>
     </Transition>
+    <Transition id="DECOY_AQRT.YILAGVENSK.2_y8_1.light" peptideRef="DECOY_YILAGVENSK">
```

Comment:
restrictTransitions() was applied previously in the unit test but is now obsolete. The additional three transitions come from a target assay originally containing 9 transitions, which were filtered to three in the original implementation.